bin/release-helper.sh

```diff
@@ -3,6 +3,7 @@
 set -e
 
 VERSION_FILE=${VERSION_FILE-VERSION}
+VERSION_PACKAGE=${VERSION_PACKAGE-*/version.py}
```

Comment:
nit: I think `VERSION_PACKAGE` is a bit of a misleading name. Maybe just `VERSION_PY`?